src/Yoti.Auth/DigitalIdentity/DigitalIdentityService.cs

```diff
@@ -125,5 +124,37 @@ internal static async Task<CreateQrResult> CreateQrCode(HttpClient httpClient, U
             }
         }
 
+        internal static async Task<GetQrCodeResult> GetQrCode(HttpClient httpClient, Uri apiUrl, string sdkId, AsymmetricCipherKeyPair keyPair, string qrCodeId)
+        {
+            Validation.NotNull(httpClient, nameof(httpClient));
+            Validation.NotNull(apiUrl, nameof(apiUrl));
+            Validation.NotNull(sdkId, nameof(sdkId));
+            Validation.NotNull(keyPair, nameof(keyPair));
+            Validation.NotNull(qrCodeId, nameof(qrCodeId));
+
+            Request shareSessionlRequest = new RequestBuilder()
```

Comment:
`fetchQrCode` or `fetchShareQrCode` or `fetchSessionQrCode` or something related to the QR?

Reply:
corrected request variable names